socorro/processor/mozilla_transform_rules.py

```diff
@@ -590,10 +591,68 @@ def __init__(self, config):
         # NOTE(willkg): These config values come from Processor2015 instance and are
         # used for lookup in product_versions
         database = config.database_class(config)
-        self.transaction = config.transaction_executor_class(
-            config,
-            database,
-        )
+        self.transaction = config.transaction_executor_class(config, database)
+
+        try:
+            sentry_dsn = self.config.sentry.dsn
+        except (AttributeError, KeyError):
+            sentry_dsn = None
+        self.sentry_dsn = sentry_dsn
```

Comment:
This is added so we can surface errors with the lookup against `crashtstats_productversion` without otherwise disrupting processing and this rule.